credentials/apps/api/v2/serializers.py

```diff
@@ -311,13 +313,41 @@ def validate_image(self, value):
         return value
 
 
+class SignatoryListField(serializers.ListField):
+    """
+    Read a docstring to exaplin this need.
```

Comment:
This docstring is a template. I thought, that you would write a real one full of context. Please do that.)

Reply:
Fixed